tools/stage_time_analysis.groovy

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 import groovy.json.JsonSlurper
```

Comment:
import not indented with the rest of the top level code

Reply:
fixed